BD-3735 Add IAM templates (#8599)
braze-inc/braze-docs: _includes/drag_and_drop/templates.md

{% if include.section == "SDK requirements" %}

> ## SDK requirements

Reviewer: Thoughts about doing this instead?

```suggestion
## Prerequisites
```

Then the H3 headings you have will be used to infer the "SDK" part?

### Minimum SDK versions

> Messages created using the drag-and-drop editor can only be sent to users on the following minimum SDK versions. See the [Prerequisites]({{site.baseurl}}/user_guide/message_building_by_channel/in-app_messages/drag_and_drop/create/#prerequisites) section of the [Creating an in-app message with drag-and-drop]({{site.baseurl}}/user_guide/message_building_by_channel/in-app_messages/drag_and_drop/create/) article for more details and nuances to be aware of.

Reviewer: ```suggestion
Messages created using the drag-and-drop editor can only be sent to users on the following minimum SDK versions. For more information, see [Creating an in-app message with drag-and-drop: Prerequisites]({{site.baseurl}}/user_guide/message_building_by_channel/in-app_messages/drag_and_drop/create/#prerequisites).
```

{% sdk_min_versions swift:5.0.0 android:8.0.0 web:2.5.0 %}

### SDK versions for text links

> If you want to include text links that do not dismiss the message, users must be on the following minimum SDK versions:

Reviewer: ```suggestion
To include text links that do not dismiss the message, the following minimum SDK versions are required:
```

(i'm _assuming_ the draft or original doc said "users" in reference to braze customers rather than end users? theres 100% chance i could be wrong though. if thats the case, please ignore this suggestion. my current understanding is that sdk version is tied to the app rather than the end-user?)
